benchmark-scripts/format_avc_mp4.sh

```diff
@@ -41,6 +42,11 @@ then
         FPS=$5
 fi
 
+if [ ! -z "$6" ]
```

Comment:
```suggestion
if [ -n "$6" ]
```
to fix reviewdog issue